common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
This commit https://github.com/edx/edx-platform/pull/12691/commits/ddecb5ad901601961f618ece02d3633332f9e71f i meant.